Day1-adding_to_k/Brandon/adding.py

```diff
@@ -0,0 +1,28 @@
+# Given a list of numbers and a number k, return whether any two numbers from the list add up to k.
+# For example, given [10, 15, 3, 7] and k of 17, return true since 10 + 7 is 17.
+# Bonus: Can you do this in one pass?
+
+from copy import deepcopy
+
+def is_k_a_sum(list_of_nums,k):
+    # Using set because of the benefit that it provides in lookup with in operator
+    sum_set = set()
```

Comment:
Set is a math term. Roughly it means that all the object inside of it are unique. Might be overkill in this function. 
Allows hashing-like lookup when you look for objects inside of it. Like is `k` in `set_sum`?